### ISSUE TITLE
Unable to import from Sticky Password. 

### ISSUE THREAD

**Issue by pmisner** [bitwarden/desktop#806]
## Describe the Bug

I am unable to import the JSON from Sticky Password. I am using version 8.2.8.15 of Sticky Password. I am using version 1.25.1 of Bitwarden on Windows 10. The update was attempted on the web. 

## Steps To Reproduce

Perform normal import routine for Sticky Password, using the recommended JSON export. 

## Expected Result

Successful import 

## Actual Result

Dialog box with Message "Data is not formatted correctly. Please check your import file and try again."

## Screenshots or Videos

n/a

## Environment

- Operating system:  Windows 10
- Installation method: [e.g. Downloaded from Bitwarden.com
- Build Version (go to "Settings" → "About"): 1.25.1 

## Additional Context

Really interested in having a good PW Manager that supports Linux well.

**Comment by djsmith85** [comment]
Hi @pmisner, thank you for opening this issue. Bitwarden's currently only supports importing `.xml` files from Sticky Password. Would you be able to provide a redacted version of your `.json` file here? Once we have an example we could look into creating a separate importer for `.json` files from Sticky Password. 

Until then, could you please try a new `.xml` export from Sticky Password and import via Bitwarden's **Sticky Password (xml)** importer.

**Comment by pmisner** [comment]
@djsmith85 I'm sorry, I meant XML. But you led me in the right direction.  I tried importing the xml into excel, got a message like this 

_Details: "Xml processing failed. Either the input is invalid or it isn't supported. (Internal error: '', hexadecimal value 0x04, is an invalid character. Line 6976, position 123.)"_

I opened the file up in notepad++ and noticed a number of what appeared to be corrupted entries. I manually removed these entries and the import was successful.  I would love it if I was able to get an error message similar to Excel's from Bitwarden, but a win's a win. 

Thank you. On to Linux!!
